src/features/targeting/contexts.js

```diff
@@ -1,17 +1,27 @@
 // @flow
 import { createContext } from 'react';
 
-import { type FtuxContextValue, type TargetingContextValue } from './types';
+import type { MessageID, MessageName } from './types';
+import type { TargetingApi } from './targeting-api';
+
+export type FtuxContextValue = {};
+
+export type TargetingContextValue = {|
+    getTargetingApi: MessageName => TargetingApi,
+    setEligibleMessageMap: ({ [MessageName]: MessageID }) => void,
+    state: any,
+|};
 
 const FtuxContext = createContext<FtuxContextValue>({});
 
 const defaultTargetingContextValue = {
     getTargetingApi: () => ({
         shouldShow: () => false,
         onSeen: () => {},
-        onClosed: () => {},
+        toClose: () => {},
```

Comment:
sorry can we make this `onClose` (i'm so sorry that my comment previously was inaccurate).